### DISCUSSION TITLE
Intellij Shortcuts for Prev / Next not working

### DISCUSSION THREAD

**Discussion by justinhandley** [community/community#6774] [Copilot Conversations] [answered]
I have tried the defaults and also remapping the shortcuts for previous / next in Intellij and nothing seems to work.  The option enter shortcut opens Copilot,and I can see multiple code options, but no matter what I map to the previous / next functions I can't get the inline code suggestions to cycle.

**Comment by jansorg** [comment, marked as answer]
Which language were you using when you tried the "Prev/Next completion" action?
Currently, only Java and Python support up to 3 completions in the editor ("Open Copilot" is different).

**Comment by justinhandley** [comment]
I was trying to use it in Javascript.  So basically if I want to see multiple options I have to open copilot.

Thanks.

**Comment by robbporto** [comment]
Same here.
